data/json/recipes/armor/suit.json

```diff
@@ -100,6 +100,23 @@
     "using": [ [ "sewing_standard", 68 ] ],
     "components": [ [ [ "fur", 48 ], [ "tanned_pelt", 8 ] ] ]
   },
+  {
+    "result": "armor_lamellar",
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_SUIT",
+    "skill_used": "fabrication",
+    "difficulty": 4,
+    "skills_required": [ "tailor", 3 ],
+    "time": 30000,
+    "book_learn": [ [ "textbook_tailor", 5 ], [ "tailor_portfolio", 5 ], [ "textbook_armschina", 4 ] ],
```

Comment:
I'd say that lamellar armour probably wouldn't be found in a tailoring textbook or portfolio. Arms machina is ok. On the other hand, it's actually a pretty straightforward thing to make if you have a guide. The minimum level to learn the recipe could be lower.